corfudb-tools/src/main/java/org/corfudb/compactor/CompactorBaseConfig.java

```diff
@@ -21,7 +21,7 @@
 public class CompactorBaseConfig {
 
     // Reduce checkpoint batch size due to disk-based nature and for smaller compactor JVM size
-    public static final int DISK_BACKED_DEFAULT_CP_MAX_WRITE_SIZE = 1 << 20;
+    public static final int DISK_BACKED_DEFAULT_CP_MAX_WRITE_SIZE = 100_000_000;
```

Comment:
Can you explain why we increase this from 1MB to ~95MB?
Since the meaning of 'write size' has changed from compressed to uncompressed, please add a comment explaining this.

Reply:
Can you explain why we increase this from 1MB to ~95MB?
  -> Since we had 100MB uncompressedSizeLimit for compactor based on this PR (https://github.com/CorfuDB/CorfuDB/pull/3778), I wanted to retain that value.

Reply:
Now that all compaction is disk-backed and the write size is for uncompressed, we can just keep one write size, instead of having `DISK_BACKED_DEFAULT_CP_MAX_WRITE_SIZE` and `DEFAULT_CP_MAX_WRITE_SIZE` which creates confusion.